resources/functionMap.php

```diff
@@ -3912,14 +3912,14 @@
 'HaruPage::textOut' => ['bool', 'x'=>'float', 'y'=>'float', 'text'=>'string'],
 'HaruPage::textRect' => ['bool', 'left'=>'float', 'top'=>'float', 'right'=>'float', 'bottom'=>'float', 'text'=>'string', 'align='=>'int'],
 'hash' => ['non-empty-string|false', 'algo'=>'string', 'data'=>'string', 'raw_output='=>'bool'],
-'hash_algos' => ['list<non-empty-string>'],
+'hash_algos' => ['non-empty-list<non-falsy-string>'],
```

Comment:
This function always returns non-empty-list and neither `'0'` nor `''`. https://www.php.net/manual/en/function.hash-algos.php